vendor_skins/Evercast/UI/data/locale/en-US.ini

```diff
@@ -529,6 +529,9 @@ Basic.MainMenu.Help.Logs="&Log Files"
 Basic.MainMenu.Help.Logs.ShowLogs="&Show Log Files"
 Basic.MainMenu.Help.Logs.UploadCurrentLog="Upload &Current Log File"
 Basic.MainMenu.Help.Logs.UploadLastLog="Upload &Last Log File"
+Basic.MainMenu.Help.CrashLogs="&Crash Logs"
+Basic.MainMenu.Help.CrashLogs.ShowLogs="&Show Log Files"
```

Comment:
What's the & for?  Reminds me of pass-by-reference

Reply:
Actually that's a Windows thing - The letter following the ampersand will be underlined, and can be used as a keyboard shortcut, e.g. Alt-F Alt-A will generally take you to (F)ile -> Save (A)s

Reply:
Ah I remember this from VB now that you mention it!

Reply:
That's where I first learned it too - VB3, IIRC :D